docs/usage/with-lock.rst.inc

```diff
@@ -57,9 +57,5 @@ running in the repo), then execute the given command as a subprocess and wait
 for its termination, release the lock and return the user command's return
 code as borg's return code.
 
-.. note::
-
-    If you copy a repository with the lock held, the lock will be present in
-    the copy. Thus, before using borg on the copy from a different host,
-    you need to use "borg break-lock" on the copied repository, because
-    Borg is cautious and does not automatically remove stale locks made by a different host.
\ No newline at end of file
+As example you can use "borg with-lock" for copy repo to another location
```

Comment:
I'm not a native speaker, but isn't it rather "... to copy a repo to another location".

Reply:
you right